### DISCUSSION TITLE
ClassNotFoundException when running allure command in aws codebuild.

### DISCUSSION THREAD

**Discussion by Hari-G-S** [allure-framework/allure2#1921] [Bug Reports]
Java versions I tried : corretto8, corretto11

In aws code build when I try to run allure command (allure -version , allure generate) its showing below error. 
Error: Could not find or load main class io.qameta.allure.CommandLine
Caused by: java.lang.ClassNotFoundException: io.qameta.allure.CommandLine

**Comment by cheshi-mantu** [comment]
@Hari-G-S Hello there!
could you please shed more light here?
how the pipleine looks like, how the tool was installed etc?
thanks
